mindsdb/utilities/cache.py

```diff
@@ -1,13 +1,17 @@
-import shelve
 import os
+import shelve
+import json
 
+import walrus
 from mindsdb.utilities.config import Config
 
+CONFIG = Config()
+
 
-class Cache:
+class LocalCache:
     def __init__(self, name, *args, **kwargs):
         self.kwargs = kwargs
-        self.cache_file = os.path.join(Config()['paths']['cache'], name)
+        self.cache_file = os.path.join(CONFIG['paths']['cache'], name)
```

Comment:
Create this config object on `__init__` on `self.config`, similar to the pattern in all other interfaces. It's very cheap to create, don't worry.

Reply:
Done